README.md

```diff
@@ -1,148 +1,431 @@
+into MSCS. Do this via:
+
+        mscs create [world] [port] <ip>
+  
+Where `world` is the world name
+(so it would be "world" from the previous example", 
+and `port` is the server port (by default, use 25565).
+`ip` is optional and will be used if you wish to create multiple worlds 
+across different servers. For now, leave it blank.
+
+Afterwards, start the server via `mscs start [world]`. 
+Then, accept the EULA, run the start command again, and you're good to go!
+
+As a last note, make sure you check `server-port` and `query.port` in 
+`server.properties` 
+to make sure it does not overlap with other servers created by the MSCS script. 
+Also ensure that `enable-query` is set to `true`.  
+If you do not have `enable-query` and a `query.port` set, 
+you will not be able to check the status of the world with the script.
+
+### Adjusting world properties
+The `mscs.properties` file can be found in every world folder 
+(for instance, if you had a world called `myWorld`, the path would be 
+`/opt/mscs/worlds/myWorld/mscs.properties`). 
+This file allows you to adjust many different properties for each world 
+you have.
+
+By default, the file only has one line in it: `mscs-enabled=true`. 
+You can add a variety of flags to this file and set them as to a true/false 
+boolean or a variable to your liking.
+
+The following flags are available:
+* mscs-enabled - Enable or disable the world server.
+* mscs-version-type - Assign the version type (release or snapshot).
+* mscs-client-version - Assign the version of the client software.
+* mscs-client-jar - Assign the .jar file for the client software.
+* mscs-client-url - Assign the download URL for the client software.
+* mscs-client-location - Assign the location of the client .jar file.
+* mscs-server-version - Assign the version of the server software.
+* mscs-server-jar - Assign the .jar file for the server software.
+* mscs-server-url - Assign the download URL for the server software.
+* mscs-server-args - Assign the arguments to the server.
+* mscs-initial-memory - Assign the initial amount of memory for the server.
+* mscs-maximum-memory - Assign the maximum amount of memory for the server.
+* mscs-server-location - Assign the location of the server .jar file.
+* mscs-server-command - Assign the command to run for the server.
+
+The following variables may be used in some of the values of the above keys:
+* $JAVA - The Java virtual machine.
+* $CURRENT_VERSION - The current Mojang Minecraft release version.
+* $CLIENT_VERSION - The version of the client software.
+* $SERVER_VERSION - The version of the server software.
+* $SERVER_JAR - The .jar file to run for the server.
+* $SERVER_ARGS - The arguments to the server.
+* $INITIAL_MEMORY - The initial amount of memory for the server.
+* $MAXIMUM_MEMORY - The maximum amount of memory for the server.
+* $SERVER_LOCATION - The location of the server .jar file.
+
+#### Example key/value pairs
+
```

Comment:
Okay, I incorporated in the recommendations

On Wed, Feb 3, 2016 at 12:55 AM, Jason M. Wood <notifications@github.com>
wrote:

> In README.md
> <https://github.com/sandain/MinecraftServerControlScript/pull/112#discussion_r51681425>
> :
>
> > +* mscs-server-location - Assign the location of the server .jar file.
> > +* mscs-server-command - Assign the command to run for the server.
> > +
> > +The following variables may be used in some of the values of the above keys:
> > +* $JAVA - The Java virtual machine.
> > +* $CURRENT_VERSION - The current Mojang Minecraft release version.
> > +* $CLIENT_VERSION - The version of the client software.
> > +* $SERVER_VERSION - The version of the server software.
> > +* $SERVER_JAR - The .jar file to run for the server.
> > +* $SERVER_ARGS - The arguments to the server.
> > +* $INITIAL_MEMORY - The initial amount of memory for the server.
> > +* $MAXIMUM_MEMORY - The maximum amount of memory for the server.
> > +* $SERVER_LOCATION - The location of the server .jar file.
> > +
> > +#### Example key/value pairs
> > +
>
> I think it is important to note that these example key/value pairs are
> also the default values and will be used if missing from mscs.properties
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sandain/MinecraftServerControlScript/pull/112/files#r51681425>
> .
>